doc/lsst.ap.pipe/pipeline-bps.rst

```diff
@@ -0,0 +1,152 @@
+
+Creating a yaml file
+====================
+
+Next we need to create a yaml file for submission. 
+Typically it will contain info about the processing campaign, desired input and output collections, resource requests for the various job types, and can include other yaml files (as we will do this this example).
+
+Here's an example .yaml file governing what gets passed to pipetask.
+It is simply a slightly modified version of the example in `ap_pipe/bps/bps_ApPipe.yaml <https://github.com/lsst/ap_pipe/blob/main/bps/bps_ApPipe.yaml>`_.
+Again, as of March 2022 this assumes you are running on one of the lsst-devl machines at NCSA.
+
+.. code-block:: yaml
+
+
+  # Path to the pipeline to run
+
+  pipelineYaml: '/path/to/my/pipeline.yaml'
+  # Format for job names and job output filenames
+  templateDataId: '{tract}_{patch}_{band}_{visit}_{exposure}_{detector}'
+  project: ApPipe
+  campaign: my_example
+  
+  # Directory where files associated with your submission, such as logs, will go.
+  submitPath: ${PWD}/bps/{outputRun}
+  computeSite: ncsapool
+  site:
+    ncsapool:
+      profile:
+        condor:
+          +Walltime: 28800
+  # Memory allocated for each quantum, in MBs
+  requestMemory: 2048
+  # CPUs to use per quantum; 1 is default.
+  requestCpus: 1
+  
+  # Arguments you would pass to pipetask run if running from the command line instead of in a bps job.
+  # Further arguments are given in the calls to 'runQuantumCommand' (see below).
+  payload:
+    # Specifies whether to run --init-only on each pipetask run before doing the real pipetask run.
+    runInit: true
+    # This will set the output collection name.
+    payloadName: my_example_name
+    # Same as -b on the command line.
+    butlerConfig: /datasets/hsc/gen3repo/rc2w50_ssw02/butler.yaml
+    # Same as -i on the command line; actual input collections may differ from what is shown here.
+    inCollection: HSC/calib,HSC/raw/all,refcats,u/diffim_sprint/templates_bestThirdSeeing
+    # Same as -o on the command line. Note: must specify outCollection with timestamp so you don't get innumerable sub-runs.
+    output : 'u/${USER}/{payloadName}'
+    # Same as -d on the command line. Here is an example of a small data query just for testing.
+    dataQuery: 'exposure IN (11690, 11692) AND detector in (49, 50)'
+  
+  # Various things for bps to customize about each pipeline task.
+  pipetask:
+    # Here you can set options to various pipeline tasks if they should run with something other than the defaults you specified above.
+    imageDifference:
+      requestMemory: 4096
+
+Notes on the yaml file
+----------------------
+* A good example of a complete pipeline yaml is `ap_pipe/pipelines/ApPipe.yaml <https://github.com/lsst/ap_pipe/blob/main/pipelines/ApPipe.yaml>`_.
+
+  * You can simply import that, or you may want to make other changes.
+* The `computeSite` option determines where your jobs will run; as of now (August 2021) the typical choice will be `ncsapool`.
```

Comment:
Where are the available options for this one documented?

Reply:
Good question! I think it is only ncsapool at the moment but I added a pointer to the ctrl_bps docs since that is where one would be setting it.